### PR TITLE
User can checkout and pay.

### PR DESCRIPTION
**Pull request by juliocj360**
User can now view shopping cart, proceed to checkout, and submit order.  #3

### CODE REVIEW

File: default.css [hunk 1]
```diff
@@ -1,3 +1,14 @@
+html,
+body,
+#result,
+.detailBox,
```
Comment (thebearingedge) [inline default.css]: `lower-kebab-case` for CSS selectors.

File: default.js [hunk 2]
```diff
@@ -76,6 +76,15 @@ var items = [
...
 var lookUp = document.getElementById('look-up');
+var aReview = document.getElementById('review-button');
+var costBeforeTaxes = 0;
+var fullCost = 0;
+var totalCost = 0;
+var bagQuantity = 0;
+var shippingCost = 0;
+var tax = 0;
+var taxes = 0;
+var checkedRadio = document.getElementById('radioButton1');
```
Comment (thebearingedge) [inline default.js]: Use `lower-kebab-case` for CSS selectors.

File: default.js [hunk 3]
```diff
@@ -235,21 +256,733 @@ function bagQtyUpdater() {
...
+  tax = costBeforeTaxes * 0.09;
+  taxes = tax.toFixed(2);
+  fullCost = costBeforeTaxes + Number(tax.toFixed(2));
+  totalCost = (Number(fullCost) + Number(fixedShipping)).toFixed(2);
   var bagCost = document.getElementById('cost-total');
-  bagCost.textContent = "Total Items: " + sum;
+  bagCost.textContent = "Total Cost: $" + totalCost;
+}
+
+function bagReview() {
```
Comment (thebearingedge) [inline default.js:281]: Add the `event` to this function's parameter list.

File: default.js [hunk 4]
```diff
@@ -235,21 +256,733 @@ function bagQtyUpdater() {
...
+  registerLoader(register);
+  shipper(deliveryOptions);
+  miniListRun();
+  submitter(submitButton);
+}
+
+function submitter(submitButton){
+  submitButton.addEventListener('click', function() {
+    nameChecker();
+//    shipRunner(0)
```
Comment (thebearingedge) [inline default.js]: Remove comments before committing.

File: default.js [hunk 5]
```diff
@@ -235,21 +256,733 @@ function bagQtyUpdater() {
...
+  outBox.className = "outBox";
+  outBox.textContent = "Much Appreciated " + Name + "! Your order is on it's way.";
+  aResult.appendChild(outBox);
+}
+
+
+function checkboxer(addressCheckbox) {
+  addressCheckbox.addEventListener('change', function() {
+    var paymentAddressBox = document.getElementById('paymentAddressBox');
+    if (addressCheckbox.checked) {
```
Comment (thebearingedge) [inline default.js]: Return opportunistically.
```js
if (addressCheckbox.checked) {
  paymentAddressBox.setAttribute('hidden', 'hidden');
} else {
  paymentAddressBox.removeAttribute('hidden');
}

if (!addressCheckbox.checked) {
  return paymentAddressBox.removeAttribute('hidden');
}
paymentAddressBox.setAttribute('hidden', 'hidden');
```

File: default.js [hunk 6]
```diff
@@ -235,21 +256,733 @@ function bagQtyUpdater() {
...
+  var itemBox = document.createElement('div');
+  itemBox.className = "reviewItemBox";
+  reviewListBox.appendChild(itemBox);
+
+  var image = document.createElement('img');
+  image.className = "miniReviewPicture";
+  image.setAttribute('src', items[index].image);
+  itemBox.appendChild(image);
+
+
```
Comment (thebearingedge) [inline default.js]: WS

File: index.html [hunk 7]
```diff
@@ -19,6 +19,7 @@
           <h3>Your Grab Bag</h3>
           <div id="qty-total">Total Items: 0</div>
```
Comment (thebearingedge) [inline index.html]: Don't abbreviate unless your life depends on it.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by juliocj360** [comment]
screen shot 2016-12-07 at 11 13 22 am

**Pull request merged**